server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -75,8 +76,9 @@ const (
 )
 
 var (
-	chunkFileSizeBytes     = flag.Int("storage.chunk_file_size_bytes", 3_000_000 /* 3 MB */, "How many bytes to buffer in memory before flushing a chunk of build protocol data to disk.")
-	enableChunkedEventLogs = flag.Bool("storage.enable_chunked_event_logs", false, "If true, Event logs will be stored separately from the invocation proto in chunks.")
+	chunkFileSizeBytes           = flag.Int("storage.chunk_file_size_bytes", 3_000_000 /* 3 MB */, "How many bytes to buffer in memory before flushing a chunk of build protocol data to disk.")
+	enableChunkedEventLogs       = flag.Bool("storage.enable_chunked_event_logs", false, "If true, Event logs will be stored separately from the invocation proto in chunks.")
+	enableFastInvocationResponse = flag.Bool("app.enable_fast_invocation_response", false, "If true, invocation responses will be filled from database values and skip parsing the events on read.")
```

Comment:
nit: consider renaming to `app.disable_invocation_event_parse_on_read`. Also consider defaulting this to true so that you don't need to explicitly configure it -- if we run into any issues we can always change the default to false and redeploy.

Reply:
fair enough, though I think we want the default for booleans to be false because of how flags work, so i'll use `app.require_invocation_event_parse_on_read`?

Reply:
Done.

Reply:
> we want the default for booleans to be false because of how flags work

hm would that mean that flipping the default to `true` wouldn't do anything? seems a bit surprising - but if that is not the case then lgtm. just want to make sure we have a way to revert this with a one-line change if needed.

Reply:
No, that works, it's just the convention that boolean flags are default false so you can flip the behavior from the default with `--flag` instead of `--flag=false`. NO big deal either way, though.